templates/colors-osx-default.stencil

```diff
@@ -0,0 +1,45 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/AliSoftware/SwiftGen
+
+import Cocoa
+
+extension NSColor {
+  convenience init(rgbaValue: UInt32) {
+    let red   = CGFloat((rgbaValue >> 24) & 0xff) / 255.0
+    let green = CGFloat((rgbaValue >> 16) & 0xff) / 255.0
+    let blue  = CGFloat((rgbaValue >>  8) & 0xff) / 255.0
+    let alpha = CGFloat((rgbaValue      ) & 0xff) / 255.0
+
+    self.init(red: red, green: green, blue: blue, alpha: alpha)
+  }
+}
+
+{% if colors %}
+{# Note: We don't use a UInt32 rawValue in this template so we can have multiple colors with the same rgbaValue #}
+enum {{enumName}} {
+  {% for color in colors %}
+  /// <span style="display:block;width:3em;height:2em;border:1px solid black;background:#{{color.rgb}}"></span>
+  /// Alpha: {{color.alpha|hexToInt|int255toFloat|percent}} <br/> (0x{{color.rgba}})
+  case {{color.name|swiftIdentifier}}
+  {% endfor %}
+
+  var rgbaValue: UInt32! {
```

Comment:
I'm wondering why `rgbaValue` is an implicitly-unwrapped-optional? I think we can drop the `!` here.

_(I mean, we have the same problem with the iOS templates, it's not specific to your addition in this PR… it's just that I'm realizing this as I'm reading the code again while reviewing your template, and it's probably from legacy when the template had a different structure but now it could be the occasion to remove them)_